common/presenters/allocations-to-table-component.js

```diff
@@ -27,11 +27,15 @@ function _byAdded(totalSlots, unfilledSlots) {
 function allocationsToTableComponent({
   showFromLocation = false,
   showRemaining = false,
+  query,
+  isSortable = true,
 } = {}) {
   const tableConfig = [
     {
       head: {
-        text: 'collections::labels.move_size',
+        isSortable,
+        key: 'moves_count',
```

Comment:
Might be good to give this a clearer name like `sortKey`? 

Reading this back without the context of the change in 2 months it wouldn't be clear that this value is only used for sorting.